### PR TITLE
button and alert

### PR DESCRIPTION
**Pull request by rahulneal**

### CODE REVIEW

File: component_styles/alerts.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    box-sizing: border-box;
+}
+.alert-icon{
+    margin-right: 0.625rem;
+    width: 1em;
+    height: 1em;
+}
+
+.alert-primary{
+    background-color: #7FD2E6;
```
Comment (arjun-computer-geek) [inline component_styles/alerts.css:18]: use color variable instead actual values

File: docs.html [hunk 2]
```diff
@@ -6,6 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="/component_styles/docs.css">
```
Comment (arjun-computer-geek) [inline docs.html:7]: a comment would be better for readability.

File: component_styles/button.css [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+}
+
+/* inactive links */
+
+.btn-disabled{
+    pointer-events: none;
+    opacity: 0.7;
+}
+/* CTA sizes */
+.btn-lg-size{
```
Comment (dcs-soni) [inline component_styles/button.css:149]: Don't need to write size in the class name. It's understood and most of the developers use `lg` to indicate the large size. So you can remove size from the class name.

File: component_styles/button.css [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+}
+
+/* floating action button */
+.btn-float-action{
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    width: 50px;
```
Comment (deepak29-git) [inline component_styles/button.css:166]: use rem instead of px

File: docs.html [hunk 5]
```diff
@@ -34,6 +36,7 @@
             <button><a href="#alert">Alert</a></button>
```
Comment (deepak29-git) [inline docs.html:36]: use class for style

**Review by deepak29-git — COMMENTED** [review]
everything is good

**Review by shashikantmaurya0007 — APPROVED** [review]
in one or two places you have used px value instead use rem and try to put descriptive comment rest looks good to me

File: component_styles/alerts.css (hunk 1 above)
Comment (rahulneal) [reply component_styles/alerts.css:18]: noted thanks!

File: docs.html (hunk 2 above)
Comment (rahulneal) [reply docs.html:7]: sure!

File: component_styles/button.css (hunk 4 above)
Comment (rahulneal) [reply component_styles/button.css:166]: ok sure.

File: component_styles/button.css (hunk 3 above)
Comment (rahulneal) [reply component_styles/button.css:149]: sure.

**Comment by rahulneal** [comment]
merging

**Pull request merged**